public/store/results/actions.ts

```diff
@@ -21,6 +21,7 @@ import {
 } from "../../util/data";
 import { getters as resultGetters } from "../results/module";
 import { getters as dataGetters } from "../dataset/module";
+import { curveCatmullRomClosed } from "d3";
```

Comment:
Accidentally included?

Reply:
yeah that's definitely weird. auto complete and auto include.